interactem-cli/interactem_cli.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import json
+import signal
+from uuid import UUID
+
+import typer
```

Comment:
you should also be importing print from rich  (see here https://typer.tiangolo.com/tutorial/printing/?h=rich#use-rich)